.github/workflows/publish.yml

```diff
@@ -8,8 +8,50 @@ on:
       - v[0-9]+.[0-9]+.[0-9]+
 
 jobs:
+  prePublishPackageTest:
+    name: Prepublish package test
+    runs-on: ubuntu-latest
+    timeout-minutes: 30 # Default is 360
```

Comment:
We have it at integration tests, as there was ones some stall experienced, but for regular unit tests I think there's no need for this setting.